README.md

```diff
@@ -1,42 +1,14 @@
-# Foxglove Studio WebSocket protocol libraries
+# Foxglove WebSocket Protocol
 
-This repository provides a protocol specification and reference implementations enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
+This repository provides a [protocol specification](docs/spec.md) and reference implementations enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
 
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+The protocol is encoding-agnostic, and supports any data format as long as the desired encoding is supported by both client and server.
 
-The following implementations are provided in this repository and as installable packages:
+Implementations are available in the following languages:
 
-| Language              | Includes         | Package name                     | Version                                                                                                                      |
-| --------------------- | ---------------- | -------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
-| Python                | server           | `foxglove-websocket`             | [![](https://shields.io/pypi/v/foxglove-websocket)](https://pypi.org/project/foxglove-websocket/)                            |
-| JavaScript/TypeScript | server + client  | `@foxglove/ws-protocol`          | [![](https://shields.io/npm/v/@foxglove/ws-protocol)](https://www.npmjs.com/package/@foxglove/ws-protocol)                   |
-| JavaScript/TypeScript | examples         | `@foxglove/ws-protocol-examples` | [![](https://shields.io/npm/v/@foxglove/ws-protocol-examples)](https://www.npmjs.com/package/@foxglove/ws-protocol-examples) |
-| C++                   | _in development_ |
-
-## Documentation
-
-- [Protocol specification](docs/spec.md)
-- [Python package README](python)
-- [JavaScript/TypeScript package README](typescript/ws-protocol)
-
-## Development
-
-### Virtualenv usage
-
-```
-pipenv install --dev
-pipenv shell
-```
-
-### Run example servers
-
-```
-python -m foxglove_websocket.examples.simple_server
-npx @foxglove/ws-protocol-examples sysmon
-```
-
-### Run example client
-
-```
-npx @foxglove/ws-protocol-examples simple-client localhost:8765
```

Comment:
You don't think the development info is useful at the high level? Should we move it to something like DEVELOPMENT.md? I know each package has some of its own info, but I think it could still be useful to provide a "how to get started developing in this repo" guide

Reply:
If anything, we should make it easier to understand where people should go next (i.e. which package to accomplish whatever task). I don't think this top-level section should duplicate README info in the packages.

Maybe we could add something like "Explore these packages below to set up a server-client connection using the Foxglove WebSocket protocol" before the table?